src/dcap_provider.cpp

```diff
@@ -67,10 +70,13 @@ static char DEFAULT_CERT_URL[] =
     "https://pck-cache-prod-webapp-eastus.azurewebsites.net/sgx/certificates";
 static std::string cert_base_url = DEFAULT_CERT_URL;
 
+// Before Azure PCK service supports caching qe identity, fetch directly from Intel server
+static char INTEL_QE_IDENTITY_URL[] = "https://api.trustedservices.intel.com/sgx/certification/v1/qe/identity";

```

Comment:
Please rename to `QE_IDENTITY_URL` because we'll want to swap this over to the Azure cache once we have support for it.

Reply:
addressed